tests/model_registry/conftest.py

```
@@ -324,7 +374,64 @@ def updated_dsc_component_state_scope_class(
 
 
 @pytest.fixture(scope="class")
-def model_registry_client(current_client_token: str, model_registry_instance_rest_endpoint: str) -> ModelRegistryClient:
+def pre_upgrade_dsc_patch(
```

Pylint (3.3.7)

[refactor] Either all return statements in a function should return an expression, or none of them should.

(R1710)